exotations/exotica_core_task_maps/src/task_map_py.cpp

```diff
@@ -75,6 +76,9 @@ PYBIND11_MODULE(exotica_core_task_maps_py, module)
     py::class_<PointToLine, std::shared_ptr<PointToLine>, TaskMap>(module, "PointToLine")
         .def_property("end_point", &PointToLine::GetEndPoint, &PointToLine::SetEndPoint);
 
+    py::class_<JointVelocityLimitConstraint, std::shared_ptr<JointVelocityLimitConstraint>, TaskMap>(module, "JointVelocityLimitConstraint")
+        .def("set_current_joint_state", &JointVelocityLimitConstraint::SetCurrentJointState);
```

Comment:
Current or previous? Vs the backward finite differencing taskmaps - just so we have uniform naming

Reply:
I was thinking current was more intuitive here however, I think you are right, it makes more sense to use previous. Will revert.